introduction.md

```diff
@@ -0,0 +1 @@
+Hello, this is me. I'm in California, dreaming about who we used to be...
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.